tests/Makefile.in

```
@@ -49,7 +49,7 @@ test:
 	@tail -n 6 $(SUMMARY) | grep ^FAILED | \
 		(read _ fail_count && \
 		if [ "$$fail_count" -ne 0 ]; then \
-			echo See $(SUMMARY) for details; \
+			echo See tests/$(SUMMARY) for details; \
```

Let's use the existing variable.

```suggestion
			echo See $(subdir)/$(SUMMARY) for details; \
```